[13:04:03] /opt/dgl/src/runtime/tensordispatch.cc:43: TensorDispatcher: dlopen failed: /public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/dgl/tensoradapter/pytorch/libtensoradapter_pytorch_1.10.2.so: cannot open shared object file: No such file or directory
Using backend: pytorch
/public1/home/scfa1517/HKS-Wasserstein-kernel/signature.py:137: ComplexWarning: Casting complex values to real discards the imaginary part
  hks[i] = np.array([np.sum(np.exp(-eigenvalues*t)*eigenvectors[i]*eigenvectors[i]) for t in sample_points])
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:517: UserWarning: Sinkhorn did not converge. You might want to increase the number of iterations `numItermax` or the regularization parameter `reg`.
  warnings.warn("Sinkhorn did not converge. You might want to "
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:484: RuntimeWarning: divide by zero encountered in true_divide
  v = b / KtransposeU
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 0
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:484: RuntimeWarning: overflow encountered in true_divide
  v = b / KtransposeU
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 26
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:485: RuntimeWarning: divide by zero encountered in true_divide
  u = 1. / nx.dot(Kp, v)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:485: RuntimeWarning: overflow encountered in true_divide
  u = 1. / nx.dot(Kp, v)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 48
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 2
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 35
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 24
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 37
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 38
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 23
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 42
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 40
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 30
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 47
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 32
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 16
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 5
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 3
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 44
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 6
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 39
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 14
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 18
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 34
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 13
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 15
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 43
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 45
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 31
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 19
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 25
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 12
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 10
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 9
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 1
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 22
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 17
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 7
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 29
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 4
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 49
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 8
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 20
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 11
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 33
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 28
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 41
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 46
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
slurmstepd: error: *** JOB 1119839 ON ia0503 CANCELLED AT 2022-03-27T15:10:41 ***
